### ISSUE TITLE
Expose the actions of MBFeedbackViewController

### ISSUE THREAD

**Issue by matthewkertis** [mapbox/mapbox-navigation-ios#2106]
We want to create a custom styled feedback controller, but the action handlers in MBFeedbackViewController are all private. Is it possible to expose these for clients to call manually?

**Mapbox Navigation SDK version:0.31.0**

**Comment by 1ec5** [comment]
Which action handlers are you referring to? If you’re trying to style FeedbackViewController, perhaps you’d want public access to the view controller’s subviews (`reportIssueLabel`, `collectionView`, `progressBar`)?

**Comment by matthewkertis** [comment]
Yeah if you can give access to the FeedbackViewController subviews so we could fully style them that would solve our issue.  The action handler i was referring to was just the ability to send the feedback from somewhere outside the FeedbackViewController.  That shouldn't be needed if we have more control over styles, thanks!

**Comment by 1ec5** [comment]
This is going to require reworking the class quite a bit. For some reason the subviews are encapsulated in closures and the feature wasn’t designed to be customizable in any way.
